iterators.tex

```diff
@@ -6349,8 +6356,8 @@
 \oldtxt{\tcode{begin} and \tcode{end} are required to be amortized constant time.}
 
 \pnum
-\newtxt{Given an object \tcode{t} of type \tcode{T}, \oldtxt{the type \tcode{T} models}
-\tcode{Range\newtxt{<T>()}} \newtxt{is satisfied} if and only if}
+\newtxt{Given a value \tcode{t} of type \tcode{T}, \tcode{Range<T>()} is satisfied
+if and only if}
```

Comment:
> "value" isn't much better.

How about "given an rvalue or lvalue `t` of type `T`, types `T`, `T&` and `T&&` model `Range` if and only if"?